src/annotator/anchoring/pdf.js

```diff
@@ -1,44 +1,70 @@
 /* global PDFViewerApplication */
 
-import seek from 'dom-seek';
-
 import RenderingStates from '../pdfjs-rendering-states';
 
-import { BrowserRange } from './range';
-import { toRange as textPositionToRange } from './text-position';
-import { TextPositionAnchor, TextQuoteAnchor } from './types';
+import { TextPosition, TextRange } from './text-range';
+import { TextQuoteAnchor } from './types';
```

Comment:
The purpose and meaning of this type is a little hard to wrap my head around. In its definition, it says

`Converts between `TextQuoteSelector` selectors and `Range` objects.`

Maybe it's that the term `Anchor` seems to be...overloaded. I don't know...

Reply:
We might be able to eliminate `TextQuoteAnchor` in future. The main purpose of the classes in `types.js` is to provide a uniform interface for converting between selectors and ranges which wraps the underlying implementation. That's useful as an interface, but I don't think it needs to have any fields. In future we could replace it with something like:

```js
class TextQuoteResolver {
  rangeFromSelector(selector) { ... }
  selectorFromRange(range) { ... }
}
```

Reply:
Yes, I think our use of the term `anchor` has exceeded the number of different things it should mean :)